Kitodo/src/main/java/org/kitodo/production/dto/ProjectDTO.java

```diff
@@ -11,13 +11,23 @@
 
 package org.kitodo.production.dto;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
+import java.util.Objects;
+
+import org.apache.logging.log4j.util.Strings;
```

Comment:
Why using a string check method from log4j instead of more common lang3.StringUtils package?

This is change should be applied on all places but I'm pointing on this only here.